### PR TITLE
Add relevant info in WordPress topic

### PR DESCRIPTION
**Pull request by alelazcano**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Changes: Add relevant info WordPress:
> - add relevant aliases
> - fix and add info in short description
> - add context and relevant info in large description

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.** ✅

### CODE REVIEW

File: topics/wordpress/index.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 ---
-aliases: wordpress-development, wordpress-site
+aliases: wordpress-development, wordpress-site, wordpress-theme, wordpress-plugin,automattic,woocommerce
```
Comment (kenyonj) [inline topics/wordpress/index.md:2]: these seem more "related" than aliases. What do you think about putting them in a related list?
```suggestion
aliases: wordpress-development, wordpress-site
related: wordpress-theme, wordpress-plugin, automattic, woocommerce
```

File: topics/wordpress/index.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 url: https://wordpress.org/
 wikipedia_url: https://en.wikipedia.org/wiki/WordPress
 ---
 WordPress is a popular content management system, used for websites and blogs. WordPress is written in PHP and MySQL and is designed to be flexible and user-friendly.
+
+It allows the addition of functionalities through plugins. These plugins make WordPress one of the most versatile and widely used platforms for forums, community social networks, ecommerce (WooCommerce), intranet websites, education and learning platforms (LMS), magazines and newspapers, etc.
+
+WordPress has its open source version at its "org" domain address, to be installed on any compatible server, and its cloud SaaS version at "com", with a managed and ready-to-use infrastructure.
+
+Some reports say that WordPress represents around 35% of the sites on the public internet.
```
Comment (kenyonj) [inline topics/wordpress/index.md:19]: This doesn't feel right to have here, almost like marketing. Can we remove?
```suggestion
```

**Review by kenyonj — CHANGES_REQUESTED** [review]

**Comment by chardso** [comment]
https://github.com/github/explore/pull/3650#issue-1640976087
This seems to have taken long,get through it such that we update the info.

**Review by ahpook — APPROVED** [review]

**Comment by ahpook** [comment]
@kenyonj i went ahead and committed the changes you suggested 👍

**Pull request merged**